src/testing/jest/jest-27-and-under/test/jest-setup-test-framework.spec.ts

```diff
@@ -15,8 +15,9 @@ describe('jest setup test framework', () => {
     });
 
     it('does nothing if there is no parent node', () => {
-      const parentNode: MockNode = undefined;
+      const parentNode: MockNode | undefined = undefined;
 
+      // the intent of this test is to ensure
```

Comment:
Ensure what? TELL ME 😂

Reply:
NEVER! I'M TAKING SECRET ONE WITH ME!

Reply:
I think this was just a backed out copy/paste that I didn't fully undo 🙃 

I'll give it the ol' delete-a-roo